### PR TITLE
chore: add js docs

### PR DESCRIPTION
**Pull request by tadejpodrekar**
* Add and update js-docs
* Replace favicon.ico on snap install
* Change case in manifest for Masca
* Restrict queryVC filter type to `"none" | "id" | "JSONPath"`
* Enable ceramic by default in dapp

### CODE REVIEW

File: packages/connector/src/index.ts [hunk 1]
```diff
@@ -23,9 +23,7 @@ const defaultSnapOrigin = 'npm:@blockchain-lab-um/masca';
  * Install and enable Masca
  *
  * Checks for existence of MetaMask Flask and installs Masca if not installed
```
Comment (andyv09) [inline packages/connector/src/index.ts:25]: Maybe add a more detailed description as this is probably the most important function for developers

File: packages/connector/src/snap.ts [hunk 2]
```diff
@@ -66,9 +71,7 @@ export async function queryVCs(
 
 /**
  * Create a VP from a list of VCs stored in Masca under the currently selected MetaMask account
```
Comment (andyv09) [inline packages/connector/src/snap.ts:73]: Create a VP from one or more VCs passed as parameter

File: packages/connector/src/snap.ts [hunk 3]
```diff
@@ -252,7 +241,6 @@ export async function setVCStore(
 
 /**
  * Get account settings (i.e. DID method, VC stores,...)
```
Comment (andyv09) [inline packages/connector/src/snap.ts:243]: ... of currently selected account

File: packages/connector/src/utils.ts [hunk 4]
```diff
@@ -59,6 +64,11 @@ export async function validateAndSetCeramicSession(
   }
 }
 
+/**
+ * Function to sign a Verifiable Presentation
```
Comment (andyv09) [inline packages/connector/src/utils.ts:68]: Maybe add `with EIP712Signature` proof format

File: packages/connector/src/utils.ts [hunk 5]
```diff
@@ -110,6 +120,11 @@ export async function signVerifiablePresentation(
   return presentation as VerifiablePresentation;
 }
 
+/**
+ * Function to sign a Verifiable Credential
```
Comment (andyv09) [inline packages/connector/src/utils.ts:124]: Same as above

File: packages/snap/src/Snap.service.ts [hunk 6]
```diff
@@ -45,6 +45,14 @@ import WalletService from './Wallet.service';
 class SnapService {
   private static origin: string;
 
+  /**
+   * Function that querries VCs from the selected VC stores.
```
Comment (andyv09) [inline packages/snap/src/Snap.service.ts:49]: queries*

File: packages/connector/src/index.ts (hunk 1 above)
Comment (tadejpodrekar) [reply packages/connector/src/index.ts:25]: Added a note explaining only MM Flask should be enabled

File: packages/snap/images/mm_icon.svg
Comment (martines3000) [inline packages/snap/images/mm_icon.svg:1]: We can remove this

File: packages/snap/src/utils/ceramicUtils.ts [hunk 7]
```diff
@@ -14,6 +14,11 @@ export const aliases = {
   tiles: {},
 };
 
+/**
+ * Function that validates a DID session.
+ * @param serializedSession - serialized DID session string
+ * @returns string - serialized DID session string
+ */
 // Should return serialized session or throw an error
```
Comment (martines3000) [inline packages/snap/src/utils/ceramicUtils.ts:22]: You can remove this line

**Review by martines3000 — APPROVED** [review]

**Pull request merged**